src/transformState.js

```diff
@@ -5,7 +5,26 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
```

Comment:
Try some desctructuring, it is really great thing to use
```suggestion
  for (const action of actions) {
  const { type, extraData, keysToRemove } = action;
    switch (type) {
      case 'clear':
        for (const key in state) {
          delete state[key];
        }
```